GSoC2019/README.md

```diff
@@ -26,7 +26,11 @@ Everything is inside the src folder.
  
 ## Local deploy
 
-Just access the index.html file in Firefox browser (in Chrome there is a restriction that requires accessing files only through HTTP or HTTPS).
+The files can be hosted locally by running the following command in the `GSoC2019/src` folder.

```

Comment:
It would be good to explain the project structure here.

Reply:
This is the explanation for the structure of the `GSoC2019/src` directory? 

Just as a sidenote, if the changes in #10 are implemented, the project structure will change a lot. Would it be better to update the documentation after the changes are made?

Reply:
I think that you can still change it now. When #10  is ready, in that PR the appropriate fixes regarding documentation should be included too.

Reply:
Agreed with @soccerdroid

Reply:
The file already has documentation regarding the structure under the `Repository structure` header. Do I need to add something or edit it in any way?

Reply:
@kgodey can you please review this.

Reply:
No, that's fine. I'll leave it to @soccerdroid and @mathemancer for final review and approval, my feedback was high-level.